WolvenKit.App/ViewModels/Tools/AssetBrowserViewModel.cs

```diff
@@ -404,29 +404,33 @@ public void UpdateSearchInArchives()
     /// <summary>
     /// Browse the left side folder tree to the folder containing the selected item. (e.g. for after searching)
     /// </summary>
-    private bool CanBrowseToFolder() => RightSelectedItem is RedFileViewModel;
+    private bool CanBrowseToFolder() => RightSelectedItem is RedFileViewModel &&
+                                        _archiveManager.GetGameFile(RightSelectedItem!.FullName) is not null;
```

Comment:
Not sure what this should fix.
`RightSelectedItem` is either set via the AB itself and so should be in the Archives or it is set via `ShowFile` which already check this itself.
Also if you are using _archiveManager.GetGameFile() you should probably exclude the project check.

Reply:
You're right, I fixed it in the next commit.

The fix itself is L433, which occasionally threw exceptions while I was browsing through ad textures